_posts/2021-12-31-a-year-in-review-2021.markdown

```diff
@@ -0,0 +1,65 @@
+- Leveraged our investment in React Native to launch the [Artsy Android App][]
+- We [opened][] 13,368 pull requests! 12,054 were [merged][] and 1,050 were [closed][]. (note: includes contributions in private repositories)
+- Responded to and resolved 29 incidents
+- Collaborated with [iTechArt][] engineers on impactful projects
+- Migrated more repositories to deploy continuously
+- Significantly improved [Core Web Vitals][] across our web surfaces
+- Answered a bunch of questions in our #dev-help Slack channel to help get each other unstuck!
+- After many false starts, we finally agreed to pull the trigger on a full auction bidding engine rewrite
+- More stable end-to-end automated testing tooling (codenamed "integrity")
+- Worked with [Applause][] to help us with external QA for our mobile apps
+- Bootstrapped a new data engineering team
+- Launched our a new recommendation model (codenamed "affinity score")
+- Developed an in-house auction results pipeline
+- Launched improvements to our recommendations (New Works for You) for the first time in a while!
+- Launched integrations with [Arta][] (shipping), [Avalara][] (tax compliance), [Algolia][] (search), [Adjust][] (analytics), [Braze][] (marketing), and [Postie][] (marketing).
+- Moved 38+ of Force apps to a new and improved V2 setup
+- Rebuilt our design system [Palette][], updated its documentation, and migrated many web and mobile surfaces to the new look
+- Rebuilt our artwork inquiry flow! Finally!
+- Migrated to a new marketing email service provider without incident (described as a project “i wouldn’t wish on my worst enemy”)
+- Added Make Offer support for artwork inquiries
+- Hired our first agile coach!
+- Tech leads are taking more ownership over the success of their teams
+- Moved a bunch of projects off of Heroku and onto our Kubernetes infrastructure
+- Invested in an improved turn-key set-up for new folks
+- Integrated our API with more third-party art management systems like [ArtBase][] and [Artsystems][]
+- Spun up a trial integration with [Split][] (experimentation and feature flagging vendor)
+
+[opened]: https://github.com/pulls?q=is%3Apr+user%3Aartsy+created%3A%3E%3D2021-01-01+-created%3A%3E%3D2022-01-01
+[merged]: https://github.com/pulls?q=is%3Apr+user%3Aartsy+created%3A%3E%3D2021-01-01+-created%3A%3E%3D2022-01-01+is%3Amerged+merged%3A%3C%3D2021-12-31
+[closed]: https://github.com/pulls?q=is%3Apr+user%3Aartsy+created%3A%3E%3D2021-01-01+-created%3A%3E%3D2022-01-01+is%3Aclosed+is%3Aunmerged
+
+[Adjust]: https://www.adjust.com
+[Algolia]: https://algolia.com
+[Applause]: https://www.applause.com
+[Arta]: https://arta.io
+[Artbase]: https://www.artbase.com
+[Artsy Android App]: https://play.google.com/store/apps/details?id=net.artsy.app
+[Artsy Engineering Radio]: https://www.buzzsprout.com/1781859
+[Artsystems]: https://www.applause.com
+[Avalara]: https://www.avalara.com
+[Braze]: https://www.braze.com
+[Core Web Vitals]: https://web.dev/vitals
```

Comment:
```suggestion
[Core Web Vitals]: https://web.dev/vitals
[Force]: https://github.com/artsy/force
```

Reply:
Addressed in https://github.com/artsy/artsy.github.io/pull/706/commits/3d46743805d457e85ba20ef940fa4a32d99005e0. Thanks!